doc/us-en/tutorials/Message-synchronize-with-iothub-through-remote-module.md

```diff
@@ -30,33 +36,91 @@ The workflow diagram is as follows.
+        path: var/db/openedge/cert
+      - name: dxc_localhub_data-V1
+        path: var/db/openedge/data
+      - name: dxc_localhub_log-V1
+        path: var/log/openedge
+      - name: dxc_localhub_client_cert-V1
+        path: var/db/openedge/certclient
+        readonly: true
+  - name: remote-iothub
+    image: hub.baidubce.com/openedgedev/openedge-remote-mqtt-linux-amd64:latest
+    replica: 1
+    mounts:
+      - name: remote-iothub-conf
+        path: etc/openedge
+        readonly: true
+      - name: remote-iothub-cert
+        path: var/db/openedge/cert
+        readonly: true
+      - name: remote-iothub-log
+        path: var/log/openedge      
+volumes:
+  # remote mqtt
+  - name: remote-iothub-conf
+    path: var/db/openedge/remote-iothub-conf
+  - name: remote-iothub-cert
+    path: var/db/openedge/remote-iothub-cert
+  - name: remote-iothub-log
+    path: var/db/openedge/remote-iothub-log
+  - name: dxc_localhub_conf-V2
+    path: var/db/openedge/dxc_localhub_conf/V2
+  - name: dxc_localhub_cert-V1
+    path: var/db/openedge/dxc_localhub_cert/V1
+  - name: dxc_localhub_log-V1
+    path: var/db/openedge/dxc_localhub_log
+  - name: dxc_localhub_data-V1
+    path: var/db/openedge/dxc_localhub_data
+```
+
+The configuration of OpenEdge Remote module is as follows:
 
 ```yaml
-name: openedge-remote-mqtt
+name: remote-iothub
 hub:
-  address: tcp://openedge-hub:1883
+  address: tcp://localhub:1883
   username: test
   password: hahaha
 remotes:
-  - name: remote
-    address: tcp://u4u6zk2.mqtt.iot.bj.baidubce.com:1883
-    clientid: 349360d3c91a4c55a57139e9085e526f
-    username: u4u6zk2/demo
-    password: XqySIYMBsjK0JkEh
+  - name: iothub
+    address: 'ssl://h7gvsuh.mqtt.iot.bj.baidubce.com:1884'
+    clientid: 11dd7422353c46fc8851ef8fb7114509
+    username: h7gvsuh/test_edge_client
+    ca: var/db/openedge/cert/ca.pem
+    cert: var/db/openedge/cert/client.pem
+    key: var/db/openedge/cert/client.key
 rules:
-  - id: rule-rcg3k6ytq
-    hub:
+  - hub:
       subscriptions:
         - topic: t1
-          qos: 1
     remote:
-      name: remote
+      name: iothub
       subscriptions:
         - topic: t2
           qos: 1
+logger:
+  path: var/log/openedge/service.log
+  level: "debug"
 ```
-
 According to the configuration of the above, it means that the Remote Module subscribes the topic `t1` from the Local Hub Module, subscribes the topic `t2` from Baidu IoT Hub. When MQTTBOX publishes a message to the topic `t1`, the Local Hub Module will receive this message and forward it to Baidu IoT Hub via Remote Module, and MQTT.fx will also receive this message(suppose MQTT.fx has already subscribed the topic `t1` before) from Baidu IoT Hub. Similarly, When we use MQTT.fx to publish a message to the topic `t2`, then Baidu IoT Hub will receive it and forward it to the Local Hub Module via Remote module. Finally, MQTTBOX will receive this message(suppose MQTTBOX has already subscribed the topic `t2` before).
```

Comment:
According to the configuration of the above, it means that the Remote module subscribes the topic `t1` from the Local Hub module, subscribes the topic `t2` from Baidu IoT Hub. When MQTTBOX publishes a message to the topic `t1`, the Local Hub module will receive this message and forward it to Baidu IoT Hub via Remote module, and MQTT.fx will also receive this message(suppose MQTT.fx has already subscribed the topic `t1` before) from Baidu IoT Hub. Similarly, When we use MQTT.fx to publish a message to the topic `t2`, then Baidu IoT Hub will receive it and forward it to the Local Hub module via Remote module. Finally, MQTTBOX will receive this message(suppose MQTTBOX has already subscribed the topic `t2` before).